src/blocks/block-devices/components/edit.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Internal dependencies
+ */
+import Inspector from './inspector';
+import Device from './device';
+import icons from './icons';
```

Comment:
What about making this a general component since it's identical to the one in `block-author-profile/components/icons.js`? Might be a good place for any general use icons we have throughout the blocks.

Reply:
Not a bad idea!